src/SFA.DAS.AANHub.Database/Scripts/AddProfileData.sql

```diff
@@ -27,8 +27,9 @@ INSERT INTO #TempProfile VALUES
 (32,'Employer Address line 2','Employer',3, 'Apprentice'),
 (33,'Employer Town or City','Employer',4, 'Apprentice'),
 (34,'Employer County','Employer',5, 'Apprentice'),
-(35,'Employer Postcode','Employer',6, 'Apprentice')
-
+(35,'Employer Postcode','Employer',6, 'Apprentice'),
+(36,'Employer address longitude','Employer',7, 'Apprentice'),
+(37,'Employer address latitude','Employer',8, 'Apprentice') 
```

Comment:
Should we not use the same capitalisation as the others?

```suggestion
(36,'Employer Address Longitude','Employer',7, 'Apprentice'),
(37,'Employer Address Latitude','Employer',8, 'Apprentice') 
```

Reply:
This text is not surfaced anywhere so it does not matter.